automobile/lidar-sensors.md

```diff
@@ -64,12 +67,51 @@ SickLms291 {
 }
 ```
 
-The `noise` field specifies the standard deviation of the gaussian noise in
+The `noise` field specifies the standard deviation of the gaussian depth noise in
```

Comment:
Shouldn't we say "depth gaussian noise" instead?

Reply:
It shouldn't, the noise applies to the depth values, and its distribution is gaussian, so gaussian serves as an adjective here and should come first.

Reply:
Thank you. Then I think this file is entirely correct related with this.